File: jmlparser-jml-tools/src/main/java/com/github/jmlparser/lint/JmlLintingConfig.java
package com.github.jmlparser.lint;

import lombok.Data;
import lombok.Getter;
import lombok.Setter;

/**
 * @author Alexander Weigl
 * @version 1 (12/29/21)
 */
@Data
public class JmlLintingConfig {
    private boolean checkNameClashes = true;
    private boolean checkMissingNames = true;

    public JmlLintingConfig() {
    }

    public boolean isDisabled(LintRule lintRule) {
        return false;
    }
}

File: tools/lint/src/main/kotlin/io/github/jmltoolkit/lint/JmlLintingConfig.kt
package io.github.jmltoolkit.lint


/**
 * @author Alexander Weigl
 * @version 1 (12/29/21)
 */
data class JmlLintingConfig(val checkNameClashes: Boolean = true, val checkMissingNames: Boolean = true) {
    fun isDisabled(lintRule: LintRule): Boolean {
        return false
    }
}
